astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -41,13 +58,25 @@ class galactocentric_frame_defaults(ScienceState):
     ``Galactocentric`` or ``Galactocentric()`` with no explicit arguments.
 
     This class controls the parameter settings by specifying a string name,
-    which can be one of:
+    with the following pre-specified options:
 
     - 'pre-v4.0': The current default value, which sets the default frame
       attribute values to their original (pre-astropy-v4.0) values.
     - 'v4.0': The attribute values as updated in Astropy version 4.0.
     - 'latest': An alias of the most recent parameter set (currently: 'v4.0')
 
+    Alternatively, user-defined parameter settings may be registered and set.
```

Comment:
Perhaps add a link here to documentation showing how to do this, or link to example below?

Reply:
Good idea. Putting a description and link to example below.